contracts/core/Registry.sol

```diff
@@ -204,11 +203,25 @@ contract Registry is AccessControl {
         roleHash = keccak256(abi.encodePacked(_identityId, _roleType));
     }
 
-    // Note: Role management - Spoke with Nate, we are going to go with single owner for now
-    // ! Who can add / remove owner for an identity? No one. It's a single owner.
-    // ? Transfer ownership will not be supported for now.
-    // - Owner can add / remove members
-    // - Single owner
+    // Note: Check with product if we can retain this function as if a role
+    // can be granted, we should have a way to revoke / transfer ownership
+    /// @notice Transfers the ownership of the identity to a new owner
+    /// @param _identityId The identityId of the identity
+    /// @param _owner New Owner
+    // function changeIdentityOwner(
```

Comment:
@nfrgosselin curious what your thoughts are

cc @codenamejason

Reply:
I think this will be okay as we can have a `changeIdentityOwner()` just create a new Identity with the new owner's address and index then generating a new anchor. The old Identity will stay intact and accessible from the original owner and index.

Reply:
Ah so that's the nice part about this. 
Based on the friday call with zak, in the current design :  
- the identityIndex and anchor are not derived from the owner
- changing the owner -> doesn't result in a change in anchor AKA the old identity stays intact

Reply:
Exactly, we don't ever lose history and is fraud resistant also.

Reply:
Marking this as resolved -> cause we are discussing this in newer comment below